lib/fluent/plugin/out_slack.rb

```diff
@@ -288,13 +300,13 @@ def build_title_payloads(chunk)
                           fields.values.map(&:title).join(' ')
                         end
 
-        {
-          channel: channel,
+        msg = {
           attachments: [{
             :fallback => fallback_text, # fallback is the message shown on popup
             :fields   => fields.values.map(&:to_h)
           }.merge(common_attachment)],
         }.merge(common_payload)
+        merge_channel(msg, channel)
```

Comment:
`msg.merge!(channel: channel) if channel` looks enough for me.

Could you write as:

```ruby
        msg = {
           attachments: [{
             :fallback => fallback_text, # fallback is the message shown on popup
             :fields   => fields.values.map(&:to_h)
           }.merge(common_attachment)],
         }
         msg.merge!(common_payload)
         msg.merge!(channel: channel) if channel
```